### PR TITLE
Add lintcss exclude in package.json

### PR DESCRIPTION
**Pull request by seanshpark**
This will add exclude folders of lintcss in package.json for external css files.

ONE-vscode-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Review by jyoungyun — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -218,7 +218,7 @@
     "pretest": "npm run compile && npm run lint && npm run lintjs && npm run lintcss && npm run linthtml",
     "lint": "eslint src --ext ts --max-warnings 0",
     "lintjs": "eslint media --ext js --max-warnings 0",
-    "lintcss": "csslint media src",
+    "lintcss": "csslint media src --exclude-list=media/Jsontracer,media/Circletracer",
```
Comment (llFreetimell) [inline package.json:221]: According to the commit message, only excluding `external` directory may be correct.

```suggestion
    "lintcss": "csslint media src --exclude-list=media/Jsontracer/external,media/Circletracer/external",
```

BTW, `media/Jsontracer` does not have `external` directory...?

Comment (seanshpark) [reply package.json:221]: css files showing warnings are in media/Jsontracer and media/Circletracer folder

Comment (seanshpark) [reply package.json:221]: `external` in commit message is from external source, not `external` folder.

Comment (llFreetimell) [reply package.json:221]: I could find `media/Jsontracer/style.css` and `media/Circletracer/style.css` in the two directories.
Those files are not external but our source code.
As the comment in both files say, they only referenced the result of external open source.

So warnings for those two files should be fixed, not ignored :)

Comment (seanshpark) [reply package.json:221]: I do not have any intension to fix these.

**Pull request closed without merging**